JitsiConferenceEventManager.js

```diff
@@ -499,6 +453,33 @@ JitsiConferenceEventManager.prototype.setupRTCListeners = function () {
                     "for not existing participant: " + from, payload);
             }
         });
+
+    if(conference.statistics) {
+        rtc.addListener(RTCEvents.CONNECTION_ICE_FAILED,
+            function (pc) {
```

Comment:
arrow function? we don't really needed for the `this` binding, but the syntax is more concise.